bib/internal/buildconfig/config.go

```diff
@@ -0,0 +1,85 @@
+package buildconfig
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/pelletier/go-toml"
+
+	"github.com/osbuild/images/pkg/blueprint"
+)
+
+type BuildConfig struct {
+	Blueprint *blueprint.Blueprint `json:"blueprint,omitempty" toml:"blueprint"`
```

Comment:
For a follow-up:
Perhaps we should import the substructs of the blueprint we support in BIB and let the structure validate and error out on the features we don't support.

Though that would make for bad error messaging probably.  It might be easier to just check for things we don't support and write a more useful error message that points to docs with a list of the things we do support.

Reply:
Sounds good! Added to the todo.